Documentation/ceph-upgrade.md

```diff
@@ -312,18 +309,29 @@ kubectl apply -f common.yaml -f crds.yaml
 
 #### **Updates for optional resources**
 
+##### Prometheus
```

Comment:
How about we change these headers to 3rd level? The headers aren't very obvious at a level lower than that.
```suggestion
### Prometheus updates (optional)
```